jest-common/src/main/java/io/searchbox/indices/script/CreateIndexedScript.java

```diff
@@ -0,0 +1,107 @@
+package io.searchbox.indices.script;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import io.searchbox.action.AbstractAction;
+import io.searchbox.action.GenericResultAbstractAction;
+
+import java.io.*;
+import java.net.URLEncoder;
+import java.nio.charset.Charset;
+
+import static io.searchbox.indices.script.ScriptLanguage.GROOVY;
+import static java.nio.charset.Charset.defaultCharset;
```

Comment:
I'd use the charset from `AbstractAction` instead (server default charsets are rarely the expected ones, at least with the other one we have a sane default).